src/features/wallet/send/SendInput.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useRef } from 'react'
+import { TouchableWithoutFeedback, TextInput } from 'react-native'
+import Box from '../../../components/Box'
+import Text from '../../../components/Text'
+import InputLock from '../../../assets/images/input-lock.svg'
+
+// TODO want to use this but need to forward refs
+// import TextInput from '../../../components/TextInput'
```

Comment:
Man. I played around with this for a while and couldn't get it to work the way I want in a reasonable amount of time 😞 .
Here's the only issue I could find regarding it.
https://github.com/Shopify/restyle/issues/40

Reply:
yeah probably should just punt on it